examples/python/data_transmission/README.cn.md

```diff
@@ -6,31 +6,31 @@
 
   在一次调用中, 客户端只能向服务器传输一次请求数据, 服务器也只能返回一次响应
 
-  `client.py - line:14 - simple_method`
+  `client.py : simple_method`
```

Comment:
Any reason for it not to be `client.py: simple_method`? [Colons generally have no space on the left, but a space on the right](https://en.wikipedia.org/wiki/Colon_(punctuation)#Usage_in_English).